app/helpers/assignment_index_helper.rb

```diff
@@ -7,17 +7,17 @@ def cal_day(day:)
 
   def cal_assignment(assignment:, day:)
     return if assignment.blank?
-    ass = 'assignment'
-    ass += '-user' if assignment.user == @current_user
+    assign = 'assignment'
+    assign += '-user' if assignment.user == @current_user
 
     if day == assignment.start_date && day == assignment.end_date
```

Comment:
I don't think that syntax works. It does if you group the expressions, but that doesn't do what you want:

```ruby
false == false == false # should be true, right? SyntaxError
(false == false) == false # OK, let's try that
true == false # Uh oh
false
```